opencensus_python_extensions_azure/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '1.0.dev0'
```

Comment:
Are we marking this as 1.0.0 (stable) as soon as we release?

Reply:
Maybe we should keep the versioning as `0.X` because other extensions might exist in this package as well, unless if you want to separate out the functions extension into its own package.

Reply:
Yes, this will become 1.0.0 once we have a new release.